mobile/lib/ui/settings/TEMP/lock_screen_option.dart

```diff
@@ -0,0 +1,230 @@
+import "package:flutter/material.dart";
+import "package:photos/core/configuration.dart";
+import "package:photos/theme/ente_theme.dart";
+import "package:photos/ui/components/captioned_text_widget.dart";
+import "package:photos/ui/components/divider_widget.dart";
+import "package:photos/ui/components/menu_item_widget/menu_item_widget.dart";
+import "package:photos/ui/components/title_bar_title_widget.dart";
+import "package:photos/ui/components/title_bar_widget.dart";
+import "package:photos/ui/components/toggle_switch_widget.dart";
+import "package:photos/ui/settings/TEMP/lock_screen_option_password.dart";
+import "package:photos/ui/settings/TEMP/lock_screen_option_pin.dart";
+import "package:photos/ui/tools/app_lock.dart";
+
+class LockScreenOption extends StatefulWidget {
+  const LockScreenOption({super.key});
+
+  @override
+  State<LockScreenOption> createState() => _LockScreenOptionState();
+}
+
+class _LockScreenOptionState extends State<LockScreenOption> {
+  final Configuration _configuration = Configuration.instance;
+  bool? appLock;
```

Comment:
Can make this non-nullable. Either use late keyword or assign an initial boolean value.